plugins/FeatureFlags/Storage/ConfigFeatureFlagStorage.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+/**
+ * Matomo - free/libre analytics platform
+ *
+ * @link    https://matomo.org
+ * @license https://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+
+namespace Piwik\Plugins\FeatureFlags\Storage;
+
+use Exception;
+use Piwik\Config;
+use Piwik\Plugins\FeatureFlags\FeatureFlagInterface;
+use Piwik\Plugins\FeatureFlags\FeatureFlagStorageInterface;
+
+class ConfigFeatureFlagStorage implements FeatureFlagStorageInterface
+{
+    private const CONFIG_FEATURE_NAME_SUFFIX = '_feature';
+    private const CONFIG_FEATURE_ENABLED_VALUE = 'enabled';
+    private const CONFIG_FEATURE_DISABLED_VALUE = 'disabled';
+
+    /**
+     * @var Config
+     */
+    private $config;
+
+    /**
+     * @internal
+     * @param Config $config
+     */
+    public function __construct(Config $config)
+    {
+        $this->config = $config;
+    }
+
+    /**
+     * @internal
+     * @param FeatureFlagInterface $feature
+     * @return bool|null
+     */
+    public function isFeatureActive(FeatureFlagInterface $feature): ?bool
+    {
+        try {
+            $featureFlagsConfig = $this->config->FeatureFlags;
+        } catch (Exception $e) {
+            return false;
+        };
+
+        $configNameForFeature = $this->getConfigNameForFeature($feature->getName());
+
+        if (!isset($featureFlagsConfig[$configNameForFeature])) {
+            return null;
+        }
+
+        $flagValue = $featureFlagsConfig[$configNameForFeature];
+
+        return $flagValue === self::CONFIG_FEATURE_ENABLED_VALUE;
+    }
+
+    /**
+     * @internal
+     * @param FeatureFlagInterface $feature
+     * @return void
+     */
+    public function disableFeatureFlag(FeatureFlagInterface $feature): void
+    {
+        if (!isset($this->config->FeatureFlags[$this->getConfigNameForFeature($feature->getName())])) {
+            return;
+        }
+
+        $this->config->FeatureFlags[$this->getConfigNameForFeature($feature->getName())] = "disabled";
```

Comment:
```suggestion
        $this->config->FeatureFlags[$this->getConfigNameForFeature($feature->getName())] = self::CONFIG_FEATURE_DISABLED_VALUE;
```

And also for the `enableFeatureFlag` case.

Perhaps even for all the mock states in the tests, so we don't have to use plain strings when there is a nice constant available.

Reply:
I've updated the implementation, I didn't however touch the tests, I don't like referencing the UUT constants in my tests (what if they are set wrong?!).

The tests are easy to read as is, and if someone changes a constant by mistake they break.